src/components/Grid/Column.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { Column } from './Column';
+
+describe('<Column />', () => {
+	it('Should be able to render', () => {
```

Comment:
shouldn't these tests show up in the storybook?

![image](https://user-images.githubusercontent.com/1710840/58539025-bfb8ad80-81f6-11e9-848b-426c62ce7f61.png)

Reply:
Right, I have to run `npm run test:output` before checking everything in. Updated!